atlp-rwanda/phantom-fe-elite
# #2 bug: Fix Setup React Router.

What does this PR do?
- Fix a bug of routing in react after setup react project .
- Added form and about components in the project which should be routed by /about and /input respectively to confirm its fix. 

Description of Task to be completed?
- Should routes correctly. at path /about, it should show message of the about. at the path /input, it should show the form. default path / should show the landing message.

How should this be manually tested?
- Clone the repo
- checkout to branch bg-fix-react-routing-2
- pull the changes by git pull
- install all packages by running npm install
- start app by running npm start
-It should automatically open the index.html into the browser and display the message showing the landing page.
At path /about it should show the about message. At path /input it should show form.
-RUN npm run build in CLI to switch into production mode

Any background context you want to provide?

What are the relevant pivotal tracker stories?
  - #2
